targets/unity-v2/Testing/Tests/Shared/Uunit/UUnitTestContext.cs

```diff
@@ -45,7 +45,11 @@ public UUnitTestContext(UUnitTestCase testInstance, Action<UUnitTestContext> tes
         public void EndTest(UUnitFinishState finishState, string resultMsg)
         {
             EndTime = DateTime.UtcNow;
-            TestResultMsg = resultMsg;
```

Comment:
Plz don't change this. Even pass messages can be vital for diagnosing failures of tests in the same suite.